core/src/main/scala/org/bitcoins/core/protocol/ln/currency/MilliSatoshis.scala

```diff
@@ -62,27 +63,63 @@ sealed abstract class MilliSatoshis extends NetworkElement {
   }
 
   def ==(ms: MilliSatoshis): Boolean = {
-    toLnCurrencyUnit == ms.toLnCurrencyUnit
+    toBigInt == ms.toBigInt
   }
 
   def !=(ms: MilliSatoshis): Boolean = {
-    toLnCurrencyUnit != ms.toLnCurrencyUnit
+    toBigInt != ms.toBigInt
   }
 
   def >=(ms: MilliSatoshis): Boolean = {
-    toLnCurrencyUnit >= ms.toLnCurrencyUnit
+    toBigInt >= ms.toBigInt
   }
 
   def >(ms: MilliSatoshis): Boolean = {
-    toLnCurrencyUnit > ms.toLnCurrencyUnit
+    toBigInt > ms.toBigInt
   }
 
   def <(ms: MilliSatoshis): Boolean = {
-    toLnCurrencyUnit < ms.toLnCurrencyUnit
+    toBigInt < ms.toBigInt
   }
 
   def <=(ms: MilliSatoshis): Boolean = {
-    toLnCurrencyUnit <= ms.toLnCurrencyUnit
+    toBigInt <= ms.toBigInt
+  }
+
+  def +(ms: MilliSatoshis): MilliSatoshis = {
+    MilliSatoshis(toBigInt + ms.toBigInt)
+  }
+
+  def -(ms: MilliSatoshis): Try[MilliSatoshis] = {
+    minus(ms)
+  }
+
+  def minus(ms: MilliSatoshis): Try[MilliSatoshis] = {
+    Try {
+      minusExc(ms)
+    }
+  }
+
+  def -!(ms: MilliSatoshis): MilliSatoshis = {
```

Comment:
What do you think of this syntax? It's meant to be an unsafe version of `-`

Reply:
I don't really like it.